src/analysis/weightsToEdgeEvaluator.test.js

```diff
@@ -16,57 +15,19 @@ describe("analysis/weightsToEdgeEvaluator", () => {
     timestampMs: 0,
   };
 
-  const fallbackNodeType = deepFreeze({
-    name: "",
-    pluralName: "",
-    prefix: NodeAddress.empty,
-    defaultWeight: 1,
-    description: "",
-  });
-
-  const srcNodeType = deepFreeze({
-    name: "",
-    pluralName: "",
-    prefix: src,
-    defaultWeight: 2,
-    description: "",
-  });
-
-  const fallbackEdgeType = deepFreeze({
-    forwardName: "",
-    backwardName: "",
-    defaultWeight: {forwards: 1, backwards: 1},
-    prefix: EdgeAddress.empty,
-    description: "",
-  });
-
   function evaluateEdge(weights: WeightsT) {
-    const evaluator = weightsToEdgeEvaluator(weights, {
-      nodeTypes: [fallbackNodeType, srcNodeType],
-      edgeTypes: [fallbackEdgeType],
-    });
+    const evaluator = weightsToEdgeEvaluator(weights);
     return evaluator(edge);
   }
 
   it("applies default weights when none are specified", () => {
-    expect(evaluateEdge(Weights.empty())).toEqual({forwards: 1, backwards: 2});
+    expect(evaluateEdge(Weights.empty())).toEqual({forwards: 1, backwards: 1});
```

Comment:
This change had me scratching my head for a bit. Assuming the previous `defaultWeight: 2` src node weight caused backwards to be `1 (default edge weight) * 2 (src node weight) * 1 (default node weight)`.